### ISSUE TITLE
[BUG] Play from TMDB Trending or other category is broken with AH2

### ISSUE THREAD

**Issue by slrwin** [jurialmunkey/skin.arctic.horizon.2#481]
### Skin section

Other

### Current Behavior

When we navigate to TMDB Helper (already configured with necessary players) trending or top rated sections in Movies or TV Shows and tried to play any show or movie by clicking the OK button in TV remote is not doing anything.

Always need to bring the content menu and select Play option with AH2 skin. With Kod's default skin I dont see this behaviour.

In Widgets created from TMDB Helper also this issue is not occuring and its only happening when we navigate to the TMBD Helper sections and try to play the item by pressing OK button in TV remote.

### Expected Behavior

When a show is focused and selected OK button from TMBD Helper trending or top rated sections should bring up the configured player to launch the show.

### Steps To Reproduce

1. Set up TMDB Helper in a TV with Android OS.
2. Set up AH2 skin as default skin.
3. Open TMDB Helper and navigate to Trending or Top rated section in Movies.
4. Select a movie and click OK button in TV remote

Log : https://zerobin.net/?bc74c84547cc449d#iwLTvlRhEGn66yim4ZK2+qILu1vB8ZYMUUpYGYtrmrI=

### Screenshots and Additional Info

Log: https://zerobin.net/?bc74c84547cc449d#iwLTvlRhEGn66yim4ZK2+qILu1vB8ZYMUUpYGYtrmrI=

### Checklist

- [X] I have searched the issues and this bug has not been reported.
- [X] I am using the most recent skin version.
- [X] I have confirmed that this bug does not occur in the default skin.

**Comment by drinfernoo** [comment]
This sounds awfully similar to #160 🤔

**Comment by slrwin** [comment]
Yes, its similar to that but its happening on My Sony tv with Kodi

**Comment by jurialmunkey** [comment]
I've never been able to reproduce this issue. The only possible thing I can think of is the fake button which sets context menu properties is blocking the action. Otherwise the skin shouldn't have any impact on playing something.

@slrwin @drinfernoo - Can you try v0.5.2 of AH2 with v5.0.2 of TMDbHelper (latest of both) and see if that fixes the problem. That removes the fake button as I've now got a better way to deal with the context menu.

**Comment by slrwin** [comment]
> I've never been able to reproduce this issue. The only possible thing I can think of is the fake button which sets context menu properties is blocking the action. Otherwise the skin shouldn't have any impact on playing something.
> 
> @slrwin @drinfernoo - Can you try v0.5.2 of AH2 with v5.0.2 of TMDbHelper (latest of both) and see if that fixes the problem. That removes the fake button as I've now got a better way to deal with the context menu.
> 
> Also please let me know what your Default Select Action setting is in Kodi Settings > Media > Videos

Sure will test it with new versions and let you know about OK button behaviour.

**Comment by slrwin** [comment]
@jurialmunkey I tested with the above mentioned versions of AH2 and TMDBHelper but I still see the same behaviour. 

This is the latest log: https://zerobin.net/?adf837f57563ceeb#D9oPpK9jV1WVZeinWOWqW05knGZmnQTxhFFBFhLXaas=

**Comment by jurialmunkey** [comment]
Can you tell me what your default select action setting is please.
Also what viewtype are you using?

**Comment by slrwin** [comment]
> Can you tell me what your default select action setting is please. Also what viewtype are you using?

I have it set to Play in Kodi setting.

**Comment by slrwin** [comment]
Sorry, could you please tell me what is viewtype?

**Comment by slrwin** [comment]
> Why are you opening MyPrograms.xml to play items? `2022-09-27 11:43:01.134 T:11767 DEBUG <general>: ------ Window Init (MyPrograms.xml) ------`
> 
> You're in the wrong window.
> 
> How are you opening Video Addons? What is your shortcut that you're using. What is the action of the shortcut or widget in the customisation dialog in skin settings?

I am not sure why its showing I am opening MyPrograms.xml.

I have home screen setup with the Widgets to Video Souces, Next, Video Addons and Programs.

I am selecting the TMDBHelper addon from the Video Addons Widget.

Its action is showing as : action://sources/video and content type as Programs.

**Comment by slrwin** [comment]
I changed the Widget content type to Videos and now the OK button is playing the shows from Trending & other sections in TMDBHelper!!

**Comment by jurialmunkey** [comment]
Content type is the window the content will open in. So for video addons and sources, you need to set this to videos. The only time you would use programs is for executable scripts. 99% of the time your content type should be videos or music.
